awsconfigs/apps/notebook-controller/deployment_patch.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: notebook-controller-deployment
+  namespace: kubeflow
+spec:
+  template:
+    spec:
+      containers:
+      - command:
```

Comment:
I dont think you anything except the name of container, can you check if specifying name of container is sufficient?

Reply:
fixed.